sdk/runtime/runenv.go

```diff
@@ -102,14 +103,26 @@ type RunEnv struct {
 	RunParams
 	*logger
 
-	unstructured chan *os.File
-	structured   chan *zap.Logger
+	MetricsPusher *push.Pusher
+	unstructured  chan *os.File
+	structured    chan *zap.Logger
 }
 
 // NewRunEnv constructs a runtime environment from the given runtime parameters.
 func NewRunEnv(params RunParams) *RunEnv {
+	containerName, _ := os.Hostname()
 	re := &RunEnv{
-		RunParams:    params,
+		RunParams: params,
+		MetricsPusher: push.New("http://prometheus-pushgateway:9091", "testground/plan").
+			Gatherer(prometheus.NewRegistry()).
+			Grouping("TestPlan", params.TestPlan).
+			Grouping("TestCase", params.TestCase).
+			Grouping("TestRun", params.TestRun).
+			Grouping("TestGroupID", params.TestGroupID).
+			Grouping("TestCaseSeq", string(params.TestCaseSeq)).
```

Comment:
I don't think we need all these groupings - they probably add a lot of overhead to the measurements.

We can remove them later though.

`TestCaseSeq` is def not needed.